storage/database/partitioned_database.go

```diff
@@ -78,7 +78,7 @@ func newPartitionedDB(dbc *DBConfig, et DBEntryType, numPartitions uint) (*parti
 		numPartitions: numPartitions, pdbBatchTaskCh: pdbBatchTaskCh}, nil
 }
 
-// batchWriteWorker executes passed batch tasks.
+// createBatchWriteWorker executes passed batch tasks.
```

Comment:
Above comment does not match actual function name.

Reply:
thanks. I reverted the change.